src/routes/cart.routes.ts

```diff
@@ -0,0 +1,35 @@
+// routes/cart.routes.ts
+import express, { Request, Response, NextFunction } from 'express';
+import { cartController } from '../controllers/cart.controller';
+import { checkUserId } from '../middleware/checkUserId';
+
+const router = express.Router();
+
+// Checkout order route
+router.post('/checkout', async (req: Request, res: Response) => {
```

Comment:
the controller can be invoked like this `router.post('/checkout', cartController.checkoutOrder)`